.changes/unreleased/Under the Hood-20221208-011018.yaml

```diff
@@ -0,0 +1,7 @@
+kind: Under the Hood
```

Comment:
Changelog entry not strictly needed, since none of the changes are to released/distributed code. But it's nbd, and always better to opt for one when you're not sure.